src/lint-files.ts

```diff
@@ -0,0 +1,17 @@
+import path from 'path';
+import kebabCase from 'kebab-case';
+
+function lintFile(file: string): string | undefined {
+  const extensionName = path.extname(file);
+  const basename = path.basename(file, extensionName);
+  const isValid = kebabCase(basename) === basename;
+
+  return isValid ? undefined : `${file} has wrong name.`;
+}
+
+export default function lintFiles(files: string[]): string[] {
+  return files.reduce((acc: string[], file: string): string[] => {
```

Comment:
`reduce` -> `flatMap`. The result should be the same, but the intention more clear.

Reply:
done